packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -482,61 +482,89 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
     async sendOps(dataModel: LoadTestDataStoreModel, config: IRunConfig) {
         const cycleMs = config.testConfig.readWriteCycleMs;
         const clientSendCount = config.testConfig.totalSendCount / config.testConfig.numClients;
+        const opsSendType = (typeof config.testConfig.opsSendType === "undefined") ?
```

Comment:
`config.testConfig.opsSendType ?? "staggeredReadWrite"` should suffice here

Reply:
Done!